### PR TITLE
[PROGRAMMES-6853] Cleaning up

### PR DESCRIPTION
**Pull request by benolot**
Ticket: https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6853

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/ValueObject/AtiAnalyticsLabels.php [hunk 1]
```diff
@@ -188,14 +188,9 @@ private function calculateProducerVariable() : string
 
     private function calculateCustomVarBrand(): ?string
     {
-        if ($this->context instanceof CoreEntity && !empty($this->context->getTleo())) {
-            $tleo = $this->context->getTleo();
-            if (!$tleo instanceof Brand) {
-                return null;
-            }
-            return $tleo->getTitle();
+        if ($this->context instanceof CoreEntity && $this->context->getTleo() && $this->context->getTleo()->isTlec()) {
```
Comment (cisco87) [inline src/ValueObject/AtiAnalyticsLabels.php]: What about```
if ($this->context instanceof CoreEntity 
    && $this->context->getTleo()
    && $this->context->getTleo()->isTlec()
){
```

**Comment by Tasiobg** [comment]
test this please

File: src/ValueObject/AtiAnalyticsLabels.php (hunk 1 above)
Comment (benolot) [reply src/ValueObject/AtiAnalyticsLabels.php]: Not sure it's worth splitting it, it's under the 120 character limit in PSR/2.

**Pull request merged**